### PR TITLE
Add Japanese translations

### PR DESCRIPTION
**Pull request by tomodachi94**
Not sure how many Japanese players you have, but this could be useful if you ever have some.

### CODE REVIEW

**Comment by Chi-Yu** [comment]
We have zero players at this point.

Does ComputerCraft even exist anymore? I think the last release was 5 years ago.

File: src/api/l10n-translations/ja/l10n_reactor [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+t_bufferTitle = 'バッファー:'
+t_outputTitle = '産出'
+t_statusTitle = '現状は'
+t_statusOnline = 'オンライン'
+t_statusOffline = 'オフライン'
+t_messagesTitle = 'メッセジ:'
+t_messagesOnline = '発電する...'
+t_messagesOffline = '待機...'
+t_messagesNoFuel = '燃料ではない'
+t_messagesBufferFull = 'Buffer full'
```
Comment (Chi-Yu) [inline src/api/l10n-translations/ja/l10n_reactor:29]: Is there a specific reason why you didn't translate these 4 strings?

File: src/api/l10n-translations/ja/l10n_reactor [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+t_startButtonTitle = 'Status Control:'
+t_startButtonStart = ' start '
+t_startButtonStop = ' stop  '
```
Comment (Chi-Yu) [inline src/api/l10n-translations/ja/l10n_reactor]: Is there a specific reason why you didn't translate these 4 strings?

Comment (tomodachi94) [reply src/api/l10n-translations/ja/l10n_reactor]: Whoops, I missed those. My bad!

**Comment by tomodachi94** [comment]
> Does ComputerCraft even exist anymore? I think the last release was 5 years ago.

ComputerCraft (the original project) is mostly dead; however, some other people made a fork called [CC: Tweaked](https://github.com/cc-tweaked/CC-Tweaked), which is much more active.

**Comment by Chi-Yu** [comment]
I cross checked your translations with Google Translate and it seems Google's grammar translation is horrible. I trust you that it doesn't sound as weird to someone who actually speaks Japanese. 😉

Add the missing ones and I can merge this, 7 years later.

I'm curious. Do you actually use these scripts somewhere?

**Comment by tomodachi94** [comment]
> I cross checked your translations with Google Translate and it seems Google's grammar translation is horrible. I trust you that it doesn't sound as weird to someone who actually speaks Japanese. wink

If you feel a pressing need, you can paste in the sentances to [Jisho.org](https://jisho.org) and verify that they are correct.

> I'm curious. Do you actually use these scripts somewhere?

I use the `strip` script occasionally when playing.

**Comment by tomodachi94** [comment]
@Chi-Yu I fixed the missing translations.

**Review by Chi-Yu — APPROVED** [review]

**Comment by Chi-Yu** [comment]
I was hoping that someone actually used the Big Reactor script, but I guess there are way more sophisticated ones out there which do a better job.

Thanks for the contribution though.

Merged.

**Pull request merged**